public/electron.js

```diff
@@ -75,15 +75,35 @@ const edit = [
           label: 'GDLauncher',
           submenu: [
             {
-              label: 'Hide',
+              label: `About GDLauncher`,
+              role: 'about'
+            },
+            { type: 'separator' },
+            {
+              label: 'Services',
+              role: 'services',
+              submenu: []
+            },
+            { type: 'separator' },
+            {
+              label: `Hide GDLauncher`,
               accelerator: 'Command+H',
-              selector: 'hide:'
+              role: 'hide'
+            },
+            {
+              label: 'Hide Others',
+              accelerator: 'Command+Alt+H',
+              role: 'hideOthers'
+            },
+            {
+              label: 'Show All',
+              role: 'unhide'
             },
             { type: 'separator' },
             {
-              label: 'Quit',
+              label: `Quit GDLauncher`,
               accelerator: 'Command+Q',
-              click: () => {
+              click() {
```

Comment:
Why changed this? If there's no need, leave it as it was

Reply:
Okay, reverted it.